File: Implementation/python/2740.py
n,m = map(int,input().split())
A = []
B = []

for _ in range(n):
    A.append(list(map(int,input().split())))
m,k = map(int,input().split())
for _ in range(m):
    B.append(list(map(int,input().split())))
result = [[0 for _ in range(k)] for _ in range(n)]
for r in range(n):
    for c in range(k):
        for i in range(m):
            result[r][c] += A[r][i] * B[i][c]
for i in range(n):
    print(" ".join(map(str,result[i])))

File: Implementation/javascript/2740.js
const input = require("fs").readFileSync("/dev/stdin").toString().trim().split("\n");

const [n, m] = input[0].split(" ").map(Number);
const [r, c] = input[n + 1].split(" ").map(Number);

const a = [];
const b = [];

for (let i = 1; i <= n; i++) {
  a.push(input[i].split(" ").map(Number));
}

for (let i = n + 2; i < n + r + 2; i++) {
  b.push(input[i].split(" ").map(Number));
}

const answer = Array.from(Array(n), () => new Array(c).fill(0));

for (let i = 0; i < n; i++) {
  for (let j = 0; j < c; j++) {
    for (let k = 0; k < m; k++) {
      answer[i][j] += a[i][k] * b[k][j];
    }
  }
}

console.log(answer.map((row) => row.join(" ")).join("\n"));
